packages/amplify-category-auth/provider-utils/awscloudformation/triggers/PostConfirmation/add-to-group.js

```diff
@@ -21,8 +21,14 @@ exports.handler = async (event, context, callback) => {
 
   try {
     await cognitoidentityserviceprovider.adminAddUserToGroup(addUserParams).promise();
-    callback(null, event);
-  } catch (e) {
-    callback(e);
+    return {
+      statusCode: 500,
```

Comment:
The statusCode should be 200 out here for a success?

Reply:
```suggestion
      statusCode: 200,
```